jedeschule/spiders/schleswig_holstein.py

```diff
@@ -20,7 +20,7 @@ def parse(self, response):
     def normalize(item: Item) -> School:
         return School(name=item.get('name'),
                       id='SH-{}'.format(item.get('id')),
-                      address=" ".join([item.get('street'), item.get('houseNumber')]),
+                      address=" ".join([item.get('street') or "", item.get('houseNumber') or ""]).strip(),
```

Comment:
I think the more pythonic version would be: 
```suggestion
                      address=" ".join([item.get('street', ""), item.get('houseNumber',  "")]).strip(),
```

but either version is fine.